### ISSUE TITLE
ARIA attribute defined inappropriately

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds.com#21]
For example, from # 39: The "aria-hidden" attribute is provided to \<table> element containing  "Let's discuss PMHC MDS.", "The Primary Mental Health...",  etc. content. As a result, screen reader users cannot access the content effectively.

Code Snippet:
> \<table role="presentation" aria-hidden="true">
> \<tbody><tr>
> \<td>
> \<h4>Let's discuss PMHC MDS.</h4>
> \<p class="large">
> The Primary Mental Health Care Minimum Data Set (PMHC MDS) is live and ready for you to > use in a way that best suits your organisation.
> \</p>
> (...)
> \</td>
> \</tr>
> \</tbody>
> \</table>

Including but not limited to - please check for this existing elsewhere:

Page Name | Accessibility Report Issue Number
------------|-----------------------------------
Communications | # 39

Numbers are the issue numbers from the Accessibility Report on the Findings worksheet at https://docs.google.com/spreadsheets/d/1fQgaUcbWK5n8gUorLlAoITfGaJC0Gqg1jzuur9wPoZg/edit#gid=585536028
